src/docs/product/sentry-basics/tracing/event-detail.mdx

```diff
@@ -64,7 +64,7 @@ The span view may be limited to one project at a time if you're on the [Team pla
 
 #### Embedding and Traversing Transactions
 
-In some cases, a span may be the parent of one or more transactions. You can identify which span has children with the expand callout. Clicking this icon will embed the first child transaction. When a transaction is embedded, the collapse callout will appear for you to hide it again.
+In some cases, a span may be the parent of one or more transactions. Spans that have children are identified by the expand icon, and clicking this icon will display the first embedded child transaction. When that transaction is displayed, the collapse icon will appear so that you can hide the transaction again.
```

Comment:
```suggestion
In some cases, a span may be the parent of one or more transactions. Spans that have children are identified by the expand icon, and clicking this icon will fetch and display the first child transaction. When that transaction is displayed, the collapse icon will appear so that you can hide the transaction again.
```